### PR TITLE
pillars of observability

### PR DESCRIPTION
**Pull request by hawkaa**

### CODE REVIEW

File: pillars-of-observability/presentation/pillars-of-observability.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+![inline](sre.jpg)
+
+^Without monitoring, you have no way to tell whether the service is even working; absent a thoughtfully designed monitoring infrastructure, you’re flying blind. Maybe everyone who tries to use the website gets an error, maybe not—but you want to be aware of problems before your users notice them.
+
+---
+
+## Observability as a part of the culture
+
+> ...provide full-stack libraries and multiple services to our internal engineering teams to monitor service health, alert on issues, support root cause investigation by providing distributed systems call traces, and support diagnosis by creating a searchable index of aggregated application/system logs.
```
Comment (guroseter) [inline pillars-of-observability/presentation/pillars-of-observability.md]: This paragraph is pretty heavy with many terms that people might not be familiar with, such as "service health" or "full stack libraries", "distributed system call traces", "searchable index". Can we try to explain the same thing with simpler terms? Maybe also add an example that everyone can relate to, or is that what is mean with "Twitter Obeservability team"

Comment (hawkaa) [reply pillars-of-observability/presentation/pillars-of-observability.md]: It is meant as an example that this is a big field and large organizations have own teams to deal with it.

File: pillars-of-observability/presentation/pillars-of-observability.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Twitter Observability team
+
+---
+
+# Logs, Metrics, and Traces
```
Comment (guroseter) [inline pillars-of-observability/presentation/pillars-of-observability.md]: Can you elaborate a bit about what is meant by "tracing", I am not sure everyone knows what this term means

File: pillars-of-observability/presentation/pillars-of-observability.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+![inline](5.png)
+
+---
+
+# Logs, Metrics, and Traces
+
+![inline](6.png)
+
```
Comment (guroseter) [inline pillars-of-observability/presentation/pillars-of-observability.md]: Nice graph!

**Comment by guroseter** [comment]
Many good points and good graphs 👍 I think it would be even better if you could relate the material even more to "daily life here". With examples we can ensure that everyone know what is meant by the terms. Suggestions: 
- Take a screen shot of a stack trace
- Take a screen shot or show in the browser how structured and unstructured logs look different in Kibana 
- Show an example of a metric that we are already monitoring in Grafana

File: pillars-of-observability/presentation/pillars-of-observability.md (hunk 2 above)
Comment (hawkaa) [reply pillars-of-observability/presentation/pillars-of-observability.md]: That is the part of the presentation :)

**Comment by hawkaa** [comment]
Thanks for feedback. I've read and received the feedback, but will mostly adjust the way I present it. It's a small primer before the hands-on workshop.

**Comment by guroseter** [comment]
Great 😃

**Pull request merged**